.gitignore

```diff
@@ -53,6 +53,7 @@ results-*.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+.pre-commit-config.yaml
```

Comment:
@Blaizzy why is this ignored?

Reply:
This was before we added pre-commit to the rep. I was using pre-commit locally.

It will be removed ASAP.

Reply:
Done ✅